com.zsmartsystems.zigbee.dongle.ember/src/main/java/com/zsmartsystems/zigbee/dongle/ember/ZigBeeDongleEzsp.java

```diff
@@ -705,12 +718,12 @@ public void passLoopbackMessages(boolean passLoopbackMessages) {
     }
 
     /**
-     * Returns an instance of the {@link EmberNcp}
+     * Returns the instance of the {@link EmberNcp}
      *
-     * @return an instance of the {@link EmberNcp}
+     * @return the instance of the {@link EmberNcp}
      */
     public EmberNcp getEmberNcp() {
-        return new EmberNcp(frameHandler);
```

Comment:
As above, I think this really needs to return a new instance, and not reuse the same instance for all users.